contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -6331,6 +6347,22 @@ bbf_ExecDropStmt(DropStmt *stmt)
 			if (schema_name != NULL)
 				logicalschema = get_logical_schema_name(schema_name, true);
 
+			/* Restrict dropping of extended stored procedures */
+			if (stmt->removeType == OBJECT_PROCEDURE && major_name)
+			{
+				int i;
+				for (i = 0; i < RESTRICTED_PROCEDURES_COUNT; i++)
```

Comment:
We should also check the schema, just procedure name comparison is not enough. End user may have procedure defined with the same name under user defined schema. Current logic does not handle it well.